cmd/arc_summary/arc_summary.py

```diff
@@ -880,7 +880,11 @@ def _tunable_summary(Kstat):
                       stderr=PIPE, shell=False, close_fds=True)
             p.wait()
 
-            description_list = p.communicate()[0].strip().split('\0')
+            # By default, Python 2 returns a string as the first element of the
+            # tuple from p.communicate(), while Python 3 returns bytes which
+            # must be decoded first
+            desc = bytes.decode(p.communicate()[0])
```

Comment:
While seems to be both python 2 and 3 compatible, I am kind of curious about how "standard" this is? I'm noticing online that others are suggesting the use of `subprocess.check_output`. I'm all for this as long as it works, just curious.

Reply:
@dinatale2 Good point. Give me a day to take a deeper look and see if it makes more sense to rewrite this more thoroughly instead of what is admittedly something of a band-aid.